structured.js

```diff
@@ -104,7 +107,10 @@
 				 operator: tree.operator.slice(0,-1),
 				 left: l,
 				 right: standardizeTree(tree.right)}};
-		} break;
+		} else {
+            r.left = standardizeTree(r.left);
+            r.right = standardizeTree(r.right);
+        } break;
```

Comment:
Are these additional calls to `standardizeTree` necessary to fix the quoting style issue?